src/pyobo/api/utils.py

```diff
@@ -46,3 +50,40 @@ def get_version(prefix: str) -> Optional[str]:
         return data["version"]
 
     return None
+
+
+def get_version_pins():
+    """Retrieve the resource version pins."""
+    try:
```

Comment:
this try/except isn't written properly - the value error could be thrown only by json.loads, so you should surround that